invenio_deposit/ext.py

```diff
@@ -58,6 +61,11 @@ def init_config(self, app):
             if k.startswith('DEPOSIT_'):
                 app.config.setdefault(k, getattr(config, k))
 
+    @staticmethod
+    def register_signals():
+        """Register signals."""
+        post_action.connect(index_record_after_publish)
```

Comment:
What if people want to use different indexers for their published records and deposits? (ex: put them in different indexes)